proposals/stage-0/code-of-conduct/FOUNDATION_CODE_OF_CONDUCT_REQUIREMENTS.md

```diff
@@ -0,0 +1,71 @@
+# Foundation Code of Conduct Requirements
+
+It is important that there be a Code of Conduct (CoC) which
+is documented and enforced for the _$FOUNDATION_ and the
+projects within it. This document defines the requirements
+for a CoC for the _$FOUNDATION_ and the projects which are part
+of the foundation.
+
+
+## Code of Conduct
+
+The _$FOUNDATION_ has adopted the
+[Contributor Covenant v1.4.1](https://www.contributor-covenant.org/version/1/4/code-of-conduct.md)
+as its code of conduct and it applies to all activities/spaces at the foundation level. In
+addition, projects joining the foundation are required to adopt this code of conduct
+within 3 months of joining the foundation.
+
+The foundation has chosen to require a common Code of Conduct to maximize its effectiveness
+at supporting projects, in their efforts to apply the Code of Conduct, as well as 
+an effective escalation process. 
```

Comment:
This sentence seems to stop making sense at the first comma.  It seems like one sentence and a second incomplete sentence bolted onto it. Maybe an editing or copy/paste error? Regardless, it is very long and has multiple things packed into it. It should be split into two sentences. I'd offer a suggestion, but I'm having trouble with it myself and my daughter is asking to watch a Strong Bad video RIGHT NOW and is getting very impatient.

Reply:
(Above comment a suggestion, not a blocking comment. Doubly so as this is apparently landing as a draft/proposal that is expected to undergo revisions.)

Reply:
I think that is a result of other comments, but I'll take another pass to try and make it better.

Reply:
Done